cypress/integration/premium/dashboard.spec.ts

```diff
@@ -0,0 +1,42 @@
+import TeamSettingsPage from "../../pages/teamSettingsPage";
+
+describe("Dashboard", () => {
+  before(() => {
+    Cypress.session.clearAllSavedSessions();
+    cy.setup();
+    cy.loginWithCySession();
+    cy.viewport(1200, 660);
+  });
+
+  after(() => {
+    cy.logout();
+  });
+
+  describe("Activity Card", () => {
+    it("displays activity when editing a teams agent options", () => {
```

Comment:
Could we build this into `teamflow.spec.ts` so we don't have to run the same UI scripts twice?

Reply:
I think we need to agree as a team on how we as organise our tests. Im seeing a couple different organisation systems especially around **free vs premium** or **user flows vs pages**.

Would you mind if we merge this as is and then come back and move it when we've decided on how we'll be organising the tests?

Reply:
100